rules/prefer-negative-index.js

```diff
@@ -0,0 +1,307 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+const isLiteralValue = require('./utils/is-literal-value');
+
+const methods = new Map([
+	[
+		'slice',
+		{
+			argumentsIndexes: [0, 1],
+			supportObjects: new Set([
+				'Array',
+				'String',
+				'ArrayBuffer',
+				'Int8Array',
+				'Uint8Array',
+				'Uint8ClampedArray',
+				'Int16Array',
+				'Uint16Array',
+				'Int32Array',
+				'Uint32Array',
+				'Float32Array',
+				'Float64Array',
+				'BigInt64Array',
+				'BigUint64Array'
+				// 'Blob'
+				// 'File'
```

Comment:
Why are these commented out?

Reply:
I found them when I search `slice`, but I guess nobody use `Blob.prototype.slice.call(...)`

Reply:
Yeah, let's drop those two.

Reply:
should I remove them? or add a comment and keep them

Reply:
Hmm, we can keep them commented out with a comment on why they're commented out, so we don't try to reintroduce them in the future.

Reply:
suggestion how to express?